pulsar-broker/src/main/java/org/apache/pulsar/broker/service/ConsistentHashingStickyKeyConsumerSelector.java

```diff
@@ -23,12 +23,12 @@
 import java.util.List;
 import java.util.Map;
 import java.util.NavigableMap;
+import java.util.NavigableSet;
 import java.util.TreeMap;
 import java.util.concurrent.CompletableFuture;
 import java.util.concurrent.locks.ReadWriteLock;
 import java.util.concurrent.locks.ReentrantReadWriteLock;
 import org.apache.pulsar.client.api.Range;
-import org.apache.pulsar.common.util.Murmur3_32Hash;
 
 /**
  * This is a consumer selector based fixed hash range.
```

Comment:
```suggestion
 * This is a consumer selector based on a fixed hash range.
```

Reply:
Thanks for the suggestion. However, I didn't write this in this PR. I'm trying to understand the original sentence and the changed sentence and it doesn't make much sense to me. It looks like this sentence was copied from HashRangeAutoSplitStickyKeyConsumerSelector. I'll remove it.

Reply:
Updated to "This is a consumer selector using consistent hashing to evenly split the number of keys assigned to each consumer."